requirements.txt

```diff
@@ -4,3 +4,4 @@ polymath
 pylint
 pep8
 pypuf_helper
+cma
```

Comment:
include an exact version here, e.g. `2.3` (this will update bugfixes but no major changes)